.storybook/main.js

```diff
@@ -7,24 +7,20 @@ const configFactory = require('../config/webpack.config');
 const sassRegex = /\.(scss|sass)$/;
```

Comment:
sorry - the formatting changes are because of prettier-on-save in sublime :grimacing: